src/Razor/src/Microsoft.VisualStudio.Editor.Razor/TelemetryReporter.cs

```diff
@@ -251,7 +261,7 @@ public void Dispose()
 
             _stopwatch.Stop();
             var values = _values.Add("eventscope.ellapsedms", _stopwatch.ElapsedMilliseconds);
-            _telemetryReporter.ReportEvent(_name, _severity, values);
+            _telemetryReporter.ReportEvent(_name, _severity, _values);
```

Comment:
This looks like a typo - this won't report the ellapsedms value that was added on the line above